app/controllers/application_controller.rb

```diff
@@ -1,3 +1,10 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  helper_method :current_user
+  def current_user
+    if session[:current_user_id]
+      @current_user ||= User.find_by(id: session[:current_user_id])
+    end
```

Comment:
Oh. I see why. Technically it won't work as is but I guess it had to be done for the client relationship on orders.